src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, ViewChild, ElementRef } from '@angular/core';
+import { MdInkBar } from '../ink-bar';
+
+@Component({
```

Comment:
Add class description for what this component does (since it's not one of the totally obvious ones like radio-button).